app/src/androidTest/java/com/github/houseorganizer/houseorganizer/TaskListActivityTest.java

```diff
@@ -109,30 +114,47 @@ public void assigneePopUpIsDisplayed() {
         onView(withText(R.string.assignees_button)).perform(click());
 
         // This line will be changed to the user of the household
```

Comment:
You might be able to remove this comment now ^^

Reply:
Thanks for the catch! Will remove it soon 👍